source/guides/clear/mixer.rst

```diff
@@ -738,32 +727,90 @@ content.
 
    #builder.conf
 
-   #VERSION 1.0
+   #VERSION 1.2
 
    [Builder]
-     CERT = "/home/clr/mix/Swupd_Root.pem"
-     SERVER_STATE_DIR = "/home/clr/mix/update"
-     VERSIONS_PATH = "/home/clr/mix"
-     YUM_CONF = "/home/clr/mix/.yum-mix.conf"
+      CERT = "/home/clr/mix/Swupd_Root.pem"
+      SERVER_STATE_DIR = "/home/clr/mix/update"
+      VERSIONS_PATH = "/home/clr/mix"
+      YUM_CONF = "/home/clr/mix/.yum-mix.conf"
+
 
    [Swupd]
-     BUNDLE = "os-core-update"
-     CONTENTURL = "<URL where the content will be hosted>"
-     VERSIONURL = "<URL where the version of the mix will be hosted>"
+      BUNDLE = "os-core-update"
```

Comment:
In my `builder.conf`, Ln 740 shows:
`BUNDLE = "os-core" [**not** "os-core-update" Please advise which is correct. ]

Reply:
it should be "os-core-update".  You're probably seen "os-core" because that is the configuration for build 3rd-party bundle.  If you just start another mix from scratch, you should see "os-core-update".

Reply:
Thanks for the clarification.